Makefile.am

```diff
@@ -34,3 +34,10 @@ tests/run-test-suite:
 	-git branch --track run-test-suite origin/run-test-suite
 	-git worktree prune
 	git worktree add $@ run-test-suite
+
+packaging:
+	git branch --track packaging origin/packaging
+	git worktree add packaging packaging
+
+docker-dist: packaging
+	make -C packaging/docker libyaml-dist
```

Comment:
`make -C $</docker libyaml-dist`

Reply:
Also please remove `packaging/` in the `distclean-local` rule.

Reply:
done